subprojects/docs/src/docs/userguide/logging.adoc

```diff
@@ -143,19 +136,23 @@ Gradle also provides integration with the Java Util Logging, Jakarta Commons Log
 
 You can replace much of Gradle's logging UI with your own. You might do this, for example, if you want to customize the UI in some way - to log more or less information, or to change the formatting. You replace the logging using the link:{groovyDslPath}/org.gradle.api.invocation.Gradle.html#org.gradle.api.invocation.Gradle:useLogger(java.lang.Object)[Gradle.useLogger(java.lang.Object)] method. This is accessible from a build script, or an init script, or via the embedding API. Note that this completely disables Gradle's default output. Below is an example init script which changes how task execution and build completion is logged.
 
-=== Example: Customizing what Gradle logs
+.Customizing what Gradle logs
+====
+include::sample[dir="userguide/initScripts/customLogger",files="init.gradle[]"]
 
-[source.multi-language-sample,groovy]
-.init.gradle
-----
-include::{samplesPath}/userguide/initScripts/customLogger/init.gradle[]
-----
+include::sample[dir="userguide/initScripts/customLoggerKts",files="customLogger.init.gradle.kts[]"]
+====
 
-.Output of **`gradle -I init.gradle build`**
+[.multi-language-text.lang-groovy]
 ----
 > gradle -I init.gradle build
 include::{samplesPath}/userguide/initScripts/customLogger/custom_logging_ui.out[]
 ----
+[.multi-language-text.lang-kotlin]
+----
+> gradle -I customLoggerKts.init.gradle.kts build
+include::{samplesPath}/userguide/initScripts/customLoggerKts/customLoggerKts.out[]
```

Comment:
In general, we want all samples to behave the same way regardless of whether groovy dsl or kotlin dsl is being used. Which means the expected output should be the _same_ for each variant of the sample, which implies there should be only one expected output for a sample that is used for both DSL language variants.